satpy/readers/olci_nc.py

```diff
@@ -184,24 +196,40 @@ def _get_solar_flux(self, band):
         return da.map_blocks(self._get_items, d_index.data,
                              solar_flux=solar_flux, dtype=solar_flux.dtype)
 
+    def getbitmask(self, quality_flags, items=None):
+        """Get the quality flags bitmask."""
+        if items is None:
+            items = DEFAULT_L1B_MASK_ITEMS
+
+        bflags = BitFlags(quality_flags, flag_list=L1B_QUALITY_FLAGS)
+
+        return reduce(np.logical_or, [bflags[item] for item in items])
```

Comment:
Is this the same?

```suggestion
        return np.array([bflags[item] for item in items]).any()
```

Reply:
No, the reduce() function return one 2D Boolean array from multiply arrays, and the result from np.array() call is a single Boolean value.

Reply:
Ok. I suppose you could do a `np.stack` then an or or any along a certain axis/dimension, but probably not worth it.